ingest-app/src/cmr/ingest/api/generic_documents.clj

```diff
@@ -163,10 +165,14 @@
     (validate-document-against-schema spec-key spec-version metadata)
     ;; The update-generic is a macro which allows for a list of URL parameters to be
     ;; passed in to be resolved by a function.
-    (tgen/update-generic request-context [provider-id native-id] (json/generate-string concept))))
+    (tgen/update-generic request-context [provider-id native-id] (json/generate-string concept))
+    {:status 204}))
```

Comment:
Why don't we want the actual message coming back to the user that the concept was ingested? Now all you are doing is providing nothing back. When ingesting collections you get back the actual collection with all its errors and warnings.
I think we should do the same. That can be done by removing {:status 204}. I think the behavior should be consistent.